season/beta_challenges/hashes.md

```diff
@@ -0,0 +1,112 @@
+# Hashing It Out
+## Movies
+Manually create a hash with the following keys/values:
+| Key | Value |
+|---|---|
+| 'Hitchcock' | ['Rear Window']|
+| 'Truffaut' | ['400 Blows'] |
+| 'Myers'  | ['Austin Powers', 'Shrek']|
+| 'fruit' | 'banana' |
+Programmatically do the following:
+```ruby 
+random = {:Hitchcock=>["Rear window"], :Truffaut=>["400 blows"], :Myers=>["Austin Powers", "Shrek"], :Fruit=>"Banana"}
+```
+1. `fruit` and `banana` aren't movies! Delete them from the hash.
+```ruby
+random.delete(:Fruit)
+```
+2. Add "The Birds" to the `Hitchcock` array.
+```ruby
+random[:Hitchcock].push("The birds")
+```
+3. Remove `Shrek`.
+```ruby
+random[:Myers].delete("Shrek")
+```
+4. Return all of the keys.
+```ruby
+random.keys    
+```
+5. Return all the values.
+```ruby
+random.values
+```
+## Quit Putin Me On
+```
+putins_brain = {
+  launch_code: "a5Mjp257GHMGH23e5qxE",
+  fav_hobby: "Riding ponies",
+  prideful: true
+}
+```
+Programmatically do the following:
+1. Return the string `"Riding ponies"`
+```ruby
+putins_brain[:fav_hobby]
+```
+2. Return the string `"a5Mjp257GHMGH23e5qxE"`
+```ruby
+putins_brain[:launch_code]
+```
+3. Add the key-value pair `obsessiveCrush: "Condoleezza Rice"`
+```ruby
+putins_brain.store(:obsessiveCrush, "Condoleezza Rice")
+```
```

Comment:
This is due to my typo, but how should you change the symbol here?

Reply:
I don't think this will work...and it's also not what I meant 😄 I meant to change it to snake case (i.e. `obsessive_crush`).